frontend/public/components/utils/resource-link.tsx

```diff
@@ -18,10 +19,18 @@ import { FlagsObject } from '../../reducers/features';
 
 const unknownKinds = new Set();
 
+// import { getActiveCluster } from '@console/internal/reducers/ui';
+
 export const resourcePathFromModel = (model: K8sKind, name?: string, namespace?: string) => {
   const { plural, namespaced, crd } = model;
+  //  const activeCluster = getActiveCluster();
+
+  let url = '';
+  // if (activeCluster) {
+  //   url += `/cluster/${activeCluster}`;
+  // }
```

Comment:
This was one approach we talked about instead of having the forwarding via ` history.pushPath`, but this only works when a user clicks on a link and doesn't automatically add the cluster to a url when a user uses a "non cluster" base url.  

In the long run, this may be the way to go, but for now just commented out.  Left the comments in so you could see the approach.

Reply:
Are these approaches mutually exclusive or could we do both?

One benefit here is if the user right-clicks the link and clicks Bookmark Link or Copy Link it will have the cluster. Otherwise the link you share with someone will use their active cluster, not yours.

Reply:
Eventually we'll want an optional `cluster` param here too to link to a resource in a different cluster.

Reply:
The changes are not mutually exclusive.  Adding this code doesn't change the behavior when clicking on a link, but I didn't think about the copy link feature.  I'll add it back in.

Reply:
I put this back in and checking for `isMultiClusterEnabled`